app/sidekiq/terms_of_use/sign_up_service_updater_job.rb

```diff
@@ -48,23 +48,49 @@ def log_updated_icn
       end
     end
 
+    def map_client
+      @map_client ||= MAP::SignUp::Service.new
+    end
+
+    def map_status
+      @map_status ||= map_client.status(icn: mpi_profile.icn)
+    end
+
+    def map_declined?
```

Comment:
You can get rid of the `map_declined?` and `map_accepted?` functions now, since they aren't used anymore